src/OrchardCore.Modules/OrchardCore.Seo/Migrations.cs

```diff
@@ -37,7 +39,14 @@ public int Create()
                     .WithSettings(new MediaFieldSettings { Multiple = false }))
             );
 
-            return 1;
+            SchemaBuilder.CreateMapIndexTable<SeoMetaPartIndex>(table => table
+                .Column<string>("PageTitle", c => c.WithLength(128)));
```

Comment:
Here you are adding a SeoMetaPartIndex table with only PageTitle. Maybe it would be worth it adding MetaKeywords, MetaDescription at least since these are also common. Then, adding MetaKeywords to Lucene indexing would make a lot more sense.